fdbserver/VersionedBTree.actor.cpp

```diff
@@ -2448,18 +2609,25 @@ class DWALPager : public IPager2 {
 
 	// Reads the most recent version of pageID, either previously committed or written using updatePage()
 	// in the current commit
-	Future<Reference<ArenaPage>> readPage(LogicalPageID pageID, bool cacheable, bool noHit = false) override {
+	Future<Reference<ArenaPage>> readPage(PagerEventReasons r,
+	                                      unsigned int l,
+	                                      LogicalPageID pageID,
+	                                      bool cacheable,
+	                                      bool noHit = false) override {
 		// Use cached page if present, without triggering a cache hit.
 		// Otherwise, read the page and return it but don't add it to the cache
+		auto& eventReasons = g_redwoodMetrics.level(l).metrics.eventReasons;
```

Comment:
`eventReasons.addEventReason(PagerEvents::pagerCacheLookup, r);`
is in every branch of this function, so just put it here after `eventReasons` is declared.